src/DynamoApplications/PathResolvers.cs

```diff
@@ -73,7 +73,7 @@ internal class CLIPathResolver : IPathResolver
         private readonly List<string> additionalNodeDirectories;
         private readonly List<string> preloadedLibraryPaths;
 
-        public CLIPathResolver(string preloaderLocation)
+        public CLIPathResolver(string preloaderLocation, string userDataFolder, string commonDataFolder)
```

Comment:
@BogdanZavu you cannot break a public signature like this in DynamoCore.

Reply:
@mjkkirschner this class is internal so there shouldn't be any breakage.
I taught it's simpler to keep just 1 constructor but lmk if you still prefer to have another one

Reply:
thanks @BogdanZavu! I overlooked that and just saw the method modifier - 1 constructor is fine with me.